x-pack/plugins/ingest_manager/server/routes/setup/index.ts

```diff
@@ -40,14 +47,19 @@ export const registerRoutes = (router: IRouter, config: IngestManagerConfigType)
     },
     getFleetStatusHandler
   );
+};
+
+export const registerRoutes = (router: IRouter, config: IngestManagerConfigType) => {
```

Comment:
> I split this function up and exported the individual register*Route functions for some now-removed tests. I left it because it seems useful is common in other Kibana plugins.

https://github.com/elastic/kibana/pull/74507#discussion_r469632746